lasagne/layers/dense.py

```diff
@@ -17,7 +17,7 @@ class DenseLayer(Layer):
     """
     lasagne.layers.DenseLayer(incoming, num_units,
     W=lasagne.init.GlorotUniform(), b=lasagne.init.Constant(0.),
-    nonlinearity=lasagne.nonlinearities.rectify, **kwargs)
+    nonlinearity=lasagne.nonlinearities.rectify, leading_axes=1, **kwargs)
```

Comment:
I'm wondering if perhaps we should call this `num_leading_axes` instead, for consistency? The name `leading_axes` implies that you can specify a list of axes, imo.